lgc/elfLinker/ElfLinker.cpp

```diff
@@ -409,6 +409,29 @@ bool ElfLinkerImpl::link(raw_pwrite_stream &outStream) {
     }
   }
 
+  // Update the size of the symbols that had code appended to them.
+  // Note that we currently cannot have the same shader get both an epilogue and a prologue.  However, if this does
```

Comment:
One day a VS will be able to have both a prologue and epilogue. Then maybe we should fix this to not rely on the order. But that's for another day...